test/unit/flux/t0001-flux.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+
+from ats.tests import AtsTest
+
+class TestFluxScheduled(unittest.TestCase):
+    @classmethod
+    def setUp():
+        self.test = AtsTest()
+
+    def test_command_list(self, test: AtsTest):
```

Comment:
Tests don't take any arguments other than ``self``.

Reply:
Sorry, I started at the bottom and read up. Again, thinking method is the way to go here.